### DISCUSSION TITLE
Bar Charts Not Displaying All Data In Dashboards

### DISCUSSION THREAD

**Discussion by tinoptilogic** [RevealBi/Reveal.Sdk#118] [General]
It makes sense why reveal is doing that, but can we apply then some logic there? For example if we are going to show only 5 records, then lets show the highest or lowest 5?

![image](https://github.com/RevealBi/Reveal.Sdk/assets/150904348/d7d38202-38f0-4c47-934f-1a9c942e9fac)

![image](https://github.com/RevealBi/Reveal.Sdk/assets/150904348/1fe19089-b434-452d-a370-3722e2b9b7c9)

**Comment by TorreyBetts** [comment]
Based on the appearance of these charts I'm guessing this is an old SDK version or you've decided to use the old charts as in the newer versions of the SDK the new charts are enabled by default. It may be worth checking to see if those solve your problem.

**Reply by tinoptilogic** [reply]
Thanks @TorreyBetts  We will soon update to the latest version and will let you know if this one is still the case.

**Reply by TorreyBetts** [reply]
One thing important you'll need to look at is how to add your license key. Newer versions of the SDK require a license key to run. Here's a [topic that describes](https://help.revealbi.io/wpf/adding-license-key) what needs done.
